File: constraint_Functions.m
% Written all the constraint functions in a matrix by converting 
% them in to the form of <= 0
function y = constraint_Functions(x)
    y = [4*x(1)^2 + x(2)^2 - 16; 3*x(1) + 5*x(2) - 4; -x(1); -x(2)];
end

File: constraint_Functions.py
import numpy as np

def constraint_functions(x):
    """
    Constraint functions in the form g(x) <= 0
    
    Args:
        x: Input vector [x1, x2]
        
    Returns:
        Array of constraint values
    """
    return np.array([4*x[0]**2 + x[1]**2 - 16, 
                     3*x[0] + 5*x[1] - 4, 
                     -x[0], 
                     -x[1]])